01-Login/src/Auth/Auth.js

```diff
@@ -17,6 +20,11 @@ export default class Auth {
     this.logout = this.logout.bind(this);
     this.handleAuthentication = this.handleAuthentication.bind(this);
     this.isAuthenticated = this.isAuthenticated.bind(this);
+    this.goTo = this.goTo.bind(this);
+
+    if (localStorage.getItem('loggedIn') === 'true') {
+      this.renewAuthentication();
+    }
   }
```

Comment:
RenewAuthentication() ends up calling checkSession() which is async, so maybe it should not be called from the constructor